.github/scripts/transform_dbt_test_results.py

```diff
@@ -1037,9 +1163,14 @@ def get_test_categories(
     conn = pyathena.connect(
         s3_staging_dir=AWS_ATHENA_S3_STAGING_DIR,
         region_name="us-east-1",
-        cursor_class=pyathena.cursor.DictCursor,
     )
-    cursor = conn.cursor()
+    # Define a regular cursor for simple queries
+    dict_cursor = conn.cursor(pyathena.cursor.DictCursor)
+    # Define a cursor with unload=True which we'll use to get more detailed
+    # column information for more complex queries. This is particularly
+    # important when requesting aggregated columns, which otherwise are
+    # deserialized incorrectly by the DictCursor
+    arrow_cursor = conn.cursor(pyathena.arrow.cursor.ArrowCursor, unload=True)
```

Comment:
**question (non-blocking):** Why not just use the `arrow_cursor` for everything?

Reply:
Good point! This was due to a slight preference I have for the `DictCursor` interface but I don't think that's worth the extra complexity of maintaining two cursors. I switched everything over to `ArrowCursor` in 94eb40115cb25ba310077d3f79f34167c6d23cd5 and confirmed it works the same by running on a subset of tests.